install/installer/pkg/components/usage/configmap_test.go

```diff
@@ -23,6 +23,7 @@ func TestConfigMap_ContainsSchedule(t *testing.T) {
 	require.JSONEq(t,
 		`{
        "controllerSchedule": "2m",
+       "contentServiceUrl": "content-service:8080",
```

Comment:
Technically, this is not a URL, just an address

Reply:
Renamed (b50982f5b)